pandas/io/formats/csvs.py

```diff
@@ -251,7 +251,7 @@ def save(self) -> None:
         try:
             # Note: self.encoding is irrelevant here
             self.writer = csvlib.writer(
-                f,
+                handleArgs.handle,  # type: ignore[arg-type]
```

Comment:
why is this type ignore?

Reply:
Argument 1 to "writer" has incompatible type "Union[IO[Any], RawIOBase, BufferedIOBase, TextIOBase, TextIOWrapper, mmap]"; expected "_Writer"  [arg-type]